python/spacewalk/satellite_tools/repo_plugins/deb_src.py

```diff
@@ -511,6 +511,10 @@ def set_download_parameters(self, params, relative_path, target_file, checksum_t
         params["minrate"] = self.minrate
         params['proxies'] = get_proxies(self.repo.proxy, self.repo.proxy_username,
                                         self.repo.proxy_password)
+        params['urlgrabber_logspec'] = None
+        with cfg_component('server.satellite') as CFG:
+            if hasattr(CFG, "urlgrabber_logspec"):
+                params["urlgrabber_logspec"] = CFG.urlgrabber_logspec
```

Comment:
[nitpick] I would simplyfy this a bit. Instead of checking the presence of an attribute and then using the attribute, I'd use the `dict`-like access that `CFG` provides.
```suggestion
        with cfg_component('server.satellite') as CFG:
            params["urlgrabber_logspec"] = CFG.get("urlgrabber_logspec")
```